hydra-node/test/Hydra/Chain/Direct/StateSpec.hs

```diff
@@ -404,6 +415,29 @@ forAllNonEmptyByronCommit action = do
           Right{} -> property False
           Left e -> action e
 
+forAllCommitWithMoreThan100ADA ::
+  (PostTxError Tx -> Property) ->
+  Property
+forAllCommitWithMoreThan100ADA action = do
+  forAll (genHydraContext maximumNumberOfParties) $ \hctx ->
+    forAll (genStInitial hctx) $ \(ctx, stInitial) ->
+      forAll genAdaOnlyUTxOOnMainnetWithAmountBiggerThan100k $ \utxo ->
```

Comment:
Could: not create this `forAllCommitWithMoreThan100ADA` and inline the generators as it's nowhere used otherwise. 

See [my PR](https://github.com/input-output-hk/hydra/pull/766) on how I extended/changed these properties here.